site-src/index.md

```diff
@@ -20,54 +20,119 @@ variety of networking use-cases.
+- **Extensible** - Gateway API allows for custom resources to be linked at 
+various layers of the API. This makes granular customization possible at the
+appropriate places within the API structure.
 
-Some other notable capabilities include …
-
-- **Shared Gateways** - They allow the sharing of load balancers and VIPs by
-permitting independent Route resources to bind to the same Gateway. This allows
-teams to share infrastructure safely without requiring direct coordination.
-- **Typed backend references** - With typed backend references Routes can
-reference Kubernetes Services, but also any kind of Kubernetes resource that is
-designed to be a Gateway backend.
-- **Cross-Namespace references** - Routes across different Namespaces can bind
-to Gateways. This allows for shared networking infrastructure despite Namespace
-segmentation for workloads.
-- **Classes** - GatewayClasses formalize types of load balancing implementations.
-These classes make it easy and explicit for users to understand what kind of
-capabilities are available as a resource model itself.
+### Why does a role-oriented API matter?
+
+Whether it’s roads, power, data centers, or Kubernetes clusters,
+infrastructure is built to be shared. In Kubernetes many individual teams
+interact with this shared infrastructure which raises a common challenge - how
+to provide flexibility to users of the infrastructure while maintaining
+control by owners of the infrastructure? The answer is in role-oriented design
+that creates a separation of concerns between roles while striking a balance
+between distributed flexibility and centralized control. 
+
+The Gateway API is a role-oriented design for Kubernetes service networking.
+It allows shared network infrastructure (hardware load balancers, cloud
+networking, cluster-hosted proxies etc) to be used by many different and
+non-coordinating teams, all bound by the policies and constraints set by
+cluster operators. The following example shows how this works in practice.
+
+The cluster operator creates a [Gateway](gateway.md) resource derived from a
+[GatewayClass](gatewayclass.md). A Gateway deploys or configures the
+underlying network resources that it represents. Through [Route binding
+policy](api-overview.md#route-binding) set on the Gateway, the operator
+permits (or denies) specific teams to bind to this Gateway and expose their
+applications through it. Centralized policies [such as
+TLS](tls.md#downstream-tls) can be enforced on the Gateway by the cluster
+operator. Meanwhile, the store and site teams run [in their own
+Namespaces](multiple-ns.md), but bind their Routes against the same shared
+Gateway, allowing them to independently control their [routing
+logic](http-routing.md). This separation of concerns allows the store team to
+manage their own [traffic splitting rollout](traffic-splitting.md) while
+leaving centralized policies and control to the cluster operators.
+
+![Gateway API Roles](./images/gateway-roles.png)
```

Comment:
This is a great diagram. Combined with the earlier diagram above, this provides a great demonstration of what this API can do.